bot.js

```diff
@@ -332,7 +341,7 @@ class Bot {
     }
     
     loadState(state, cb, eb) { /* {{{ */
-        if (this.dead) {
+        if (this.isUnusable()) {
             if (config.DEBUG) { this.log("Attempting to load dead bot") }
```

Comment:
Think it's okay here and below at 399 to still log 'dead' bot instead of 'unusable' bot?